### PR TITLE
[1LP][RFR] custom css manual test steps update

### PR DESCRIPTION
**Pull request by psimovec**

### CODE REVIEW

File: cfme/tests/test_db_migrate_manual.py [hunk 1]
```diff
@@ -288,19 +288,23 @@ def test_update_webui_custom_css():
...
-               check customization"s still work
+        initialEstimate: 1/15h
+
+        setup:  1. provision appliance
+                2. ssh to the appliance
+                3. change custom CSS file - you can run command:
+        vmdb && cd public &&\
+        wget 'https://image.spreadshirtmedia.com/image-server/v1/designs/m11541836-6,width=100.png'\
+        && echo "html {cursor: url('m11541836-6,width=100.png'), pointer;}" >> custom.css
+                4. open webui and check css changed you made (different cursor)
```
Comment (jaryn) [inline cfme/tests/test_db_migrate_manual.py]: The steps are not complete. After the step 4 we need to the
- add repo file to /etc/yum.repos.d/
- add RHSM settings
- update through webui
- check webui is available
- check customization"s still work

**Comment by jaryn** [comment]
As there are some steps needed to fulfil the test, it is not good idea to change the cursor as the one you picked is damn imprecise as ... it was a potato. !@#$% (: Let's change some colours.

File: cfme/tests/test_db_migrate_manual.py [hunk 2]
```diff
@@ -288,19 +288,23 @@ def test_update_webui_custom_css():
...
         caseimportance: medium
-        initialEstimate: 1/6h
-        setup: provision appliance
-               add custom css file
-               add repo file to /etc/yum.repos.d/
-               add RHSM settings
-               update through webui
-               check webui is available
-               check customization"s still work
+        initialEstimate: 1/15h
```
Comment (jaryn) [inline cfme/tests/test_db_migrate_manual.py]: I also would keep the time as it was as there is a lot of time needed for putting the repo file on the appliance and doing the update

File: cfme/tests/test_db_migrate_manual.py [hunk 3]
```diff
@@ -288,19 +288,25 @@ def test_update_webui_custom_css():
...
         initialEstimate: 1/6h
-        setup: provision appliance
-               add custom css file
-               add repo file to /etc/yum.repos.d/
-               add RHSM settings
-               update through webui
-               check webui is available
-               check customization"s still work
+
+        setup:  1. provision appliance
```
Comment (john-dupuy) [inline cfme/tests/test_db_migrate_manual.py]: As far as I can tell, this should be part of the `testSteps` and not the `setup`? If so, please move to `testSteps` and give corresponding `expectedResults`

File: cfme/tests/test_db_migrate_manual.py [hunk 4]
```diff
@@ -288,19 +288,29 @@ def test_update_webui_custom_css():
...
+        testSteps:
+                1. ssh to the appliance
+                2. change custom CSS file - you can run command:
+                    echo 'body {background-color: #FF00EF;}' >> /var/www/miq/vmdb/public/custom.css
+                3. add repo file to /etc/yum.repos.d/
+                4. add RHSM settings
+                5. update through webui
+
+        expectedResults:
+                webui is available
```
Comment (john-dupuy) [inline cfme/tests/test_db_migrate_manual.py]: @psimovec These expected results must line up with the `testSteps`

see e.g.

https://github.com/ManageIQ/integration_tests/blob/65501a812cecb79720712a390554f6e4ccf3e3cb/cfme/tests/automate/custom_button/test_service_objects.py#L589-L620

File: cfme/tests/test_db_migrate_manual.py [hunk 5]
```diff
@@ -288,19 +288,32 @@ def test_update_webui_custom_css():
...
-               update through webui
-               check webui is available
-               check customization"s still work
+
+        setup:     provision appliance
+
+        testSteps:
+                1. ssh to the appliance
+                2. change custom CSS file - you can run command:
+                    echo 'body {background-color: #FF00EF;}' >> /var/www/miq/vmdb/public/custom.css
```
Comment (mshriver) [inline cfme/tests/test_db_migrate_manual.py]: Looking forward to you automating this too :) There's an `ssh_client` on the appliance that makes this nice and easy.

**Pull request merged**